fdbserver/GrvProxyTagThrottler.actor.cpp

```diff
@@ -202,16 +210,14 @@ void GrvProxyTagThrottler::releaseTransactions(double elapsed,
 		}
 	}
 
-	// End release windows for queues with valid rateInfo
```

Comment:
The changed comment here says that this function ends release windows for all tag queues, but it calls `queue.endReleaseWindow` which is only doing so for those with valid `rateInfo`. Does the first for loop count as ending release windows?

Reply:
Yeah, I'm counting the first loop as ending the release window. It just happens to be a noop in the implementation if `rateInfo` is not present.